src/EnergyPlus/HybridUnitaryAirConditioners.cc

```diff
@@ -0,0 +1,654 @@
+		using CurveManager::CurveValue;
+		using HybridEvapCoolingModel::CMode;
+
+		Array1D< Model > ZoneHybridUnitaryAirConditioner;
+		int NumZoneHybridEvap(0);
+		Array1D_bool CheckZoneHybridEvapName;
+		bool GetInputZoneHybridEvap(true);
+		//ConfigFile* pConfig = new ConfigFile;
+		//Begin routines for zone HVAC Hybrid Evaporative cooler unit
+		//_______________________________________________________________________________________________________________________
+		//***************
+		void
+			SimZoneHybridUnitaryAirConditioners(
+				std::string const & CompName, // name of the packaged terminal heat pump
+				int const ZoneNum, // number of zone being served
+				Real64 & SensibleOutputProvided, // sensible capacity delivered to zone
+				Real64 & LatentOutputProvided, // Latent add/removal  (kg/s), dehumid = negative
+				int & CompIndex // index to zone hvac unit
+				)
+		{
+			using InputProcessor::FindItemInList;
+			using General::TrimSigDigits;
+			// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+			int CompNum;
+			bool errorsfound = false;
+			if (GetInputZoneHybridEvap) {
+				GetInputZoneHybridUnitaryAirConditioners(errorsfound);
+				GetInputZoneHybridEvap = false;
+			}
+
+			if (CompIndex == 0) {
+				CompNum = FindItemInList(CompName, ZoneHybridUnitaryAirConditioner);
+				if (CompNum == 0) {
+					ShowFatalError("SimZoneHybridUnitaryAirConditioners: Zone evaporative cooler unit not found.");
+				}
+				CompIndex = CompNum;
+			}
+			else {
+				CompNum = CompIndex;
+				if (CompNum < 1 || CompNum > NumZoneHybridEvap) {
+					ShowFatalError("SimZoneHybridUnitaryAirConditioners: Invalid CompIndex passed=" + TrimSigDigits(CompNum) + ", Number of units =" + TrimSigDigits(NumZoneHybridEvap) + ", Entered Unit name = " + CompName);
+				}
+				if (CheckZoneHybridEvapName(CompNum)) {
+					if (CompName != ZoneHybridUnitaryAirConditioner(CompNum).Name) {
+						ShowFatalError("SimZoneHybridUnitaryAirConditioners: Invalid CompIndex passed=" + TrimSigDigits(CompNum) + ", Unit name=" + CompName + ", stored unit name for that index=" + ZoneHybridUnitaryAirConditioner(CompNum).Name);
+					}
+					CheckZoneHybridEvapName(CompNum) = false;
+				}
+			}
+			try
+			{
+				InitZoneHybridUnitaryAirConditioners(CompNum, ZoneNum);
+			}
+			catch (int e)
+			{
+				cout << "An exception occurred in InitZoneHybridUnitaryAirConditioners. Exception Nr. " << e << '\n';
+				return;
+			}
+			try
+			{
+				CalcZoneHybridUnitaryAirConditioners(CompNum, ZoneNum, SensibleOutputProvided, LatentOutputProvided);
+			}
+			catch (int e)
+			{
+				cout << "An exception occurred in CalcZoneHybridUnitaryAirConditioners. Exception Nr. " << e << '\n';
+				return;
+			}
+			try
+			{
+				ReportZoneHybridUnitaryAirConditioners(CompNum);
+			}
+			catch (int e)
+			{
+				cout << "An exception occurred in ReportZoneHybridUnitaryAirConditioners. Exception Nr. " << e << '\n';
+				return;
+			}
+		}
+		Model* HandelToHybridUnitaryAirConditioner(int UnitNum)
```

Comment:
ok sorry, Im perhaps the worlds worst speller and there is no spell check in this thing. :-) 

Thanks for catching, fixed